blog/2015-05-27-so-you-want-to-transform-part-1.adoc

```diff
@@ -0,0 +1,211 @@
+:page-layout: blog
```

Comment:
you should just delete the empty space, not the title ;) I've added it in before I pushed.